robot/web/web.go

```diff
@@ -362,8 +381,10 @@ func (svc *webService) Close(ctx context.Context) error {
 	defer svc.mu.Unlock()
 	svc.stopWeb()
 	var err error
-	if svc.modServer != nil {
-		err = svc.modServer.Stop()
+	for _, srv := range []rpc.Server{svc.tcpModServer, svc.unixModServer} {
+		if srv != nil {
+			err = srv.Stop()
```

Comment:
consider making this a multi error? doesn't super matter though, if both servers fail, getting the error from either is probably sufficient